src/rosdep2/installers.py

```diff
@@ -540,36 +540,36 @@ def install_resolved(self, installer_key, resolved, simulate=False, interactive=
         if simulate:
             print("#[%s] Installation commands:"%(installer_key))
             for sub_command in command:
-                if type(sub_command[0]) is list:
-                    l = len(sub_command)
-                    for i in range(l):
-                        print('  %s (alternative %d/%d)'%(' '.join(sub_command[i]), i+1,l))
+                if isinstance(sub_command[0], list):
+                    len = len(sub_command)
+                    for cmd, i in enumerate(l):
```

Comment:
Yeah, sorry, that's what I get for relying on my local editor so much and then trying to edit with the web ui.